src/Command/ConfigureCommand.php

```diff
@@ -27,6 +27,8 @@
 
 final class ConfigureCommand extends BaseCommand
 {
+    const NONINTERACTIVE_MODE_ERROR = 'Infection config generator requires an interactive mode.';
```

Comment:
do we need to expose such messages via a constant? It feels a bit overkill plus right now they are exposed as public constants. As we're in 7.0 we cannot use `private const` but we could mark them with `@private`.

Reply:
How about if mark the entire class as private?